CONTRIBUTING.md

```diff
@@ -4,15 +4,17 @@
 ## Installation for Local Development
 
 Requirements
-* Python 3.7
-* pip3
-* VirtualEnv
+* Python 3
+* pip
 
+```shell
+%> python -V                   # Check your version of python
+%> python -m venv .venv        # setup a virtual env in your cloned repo
+%> source ./venv/bin/activate  # use the virtual env
```

Comment:
I think maybe this should be `source ./.venv/bin/activate` since lin 12 has `.venv` as the name.  This would make the resulting `(venv)` bits read `(.venv)`.

Reply:
derp! thanks :)